.github/workflows/build-and-test.yml

```diff
@@ -35,4 +35,4 @@ jobs:
         run: docker build . --file Dockerfile --tag "zeroforcing:${SHA}"
       - name: Test ZeroForcing code
         run: |
-            docker run --rm "zeroforcing:${SHA}" /usr/bin/sage --python3 -m pytest --profile
+            docker run --rm "zeroforcing:${SHA}" 'sage -pip install pytest && cd zeroforcing && sage --python3 -m pytest --profile'
```

Comment:
Make this less hacky. We should be able to do `sage --python3 setup.py develop` or something and it should be able to install pytest automatically.